Makefile

```diff
@@ -115,8 +117,9 @@ set-image-controller: manifests kustomize
 
 # Deploy controller in the current Kubernetes context, configured in ~/.kube/config
 .PHONY: deploy
-deploy: set-image-controller
+deploy: set-image-controller cert-manager
```

Comment:
is there a reason to install cert-manager in the same step? Couldnt it be annoying on a system running without cert-manager?

Reply:
Ok, I'll remove this dependency.